src/bundle/Resources/public/js/scripts/helpers/content.type.helper.js

```diff
@@ -67,6 +67,18 @@ const getContentTypeName = (contentTypeIdentifier) => {
     return contentTypesDataMap[contentTypeIdentifier].name;
 };
 
+const getContentTypeByIdentifier = (contentTypeIdentifier) => {
```

Comment:
```suggestion
const getContentTypeData = (contentTypeIdentifier) => {
```
I think this will be more inline with the pattern here.

Reply:
question is shouldn't we change pattern in general, to tell explicitly that it's by identifier

Reply:
maybe in 5.0